### DISCUSSION TITLE
How can I change the order in the head tag?

### DISCUSSION THREAD

**Discussion by Vivi-Ornitier** [nuxt/nuxt#16381] [Questions]
I want `<meta charset="utf-8">` to be at the top inside the head tag.
Because `entry.mjs` is garbled in Safari.
Since I am Japanese, Safari's default encoding is Shift-JIS.
If I change the encoding to UTF-8 it works fine. However, it is difficult to correct it to users.

How can I change the order in the head tag?

Now.
```
<head>
<link rel="modulepreload" as="script" crossorigin="" href="/_nuxt/@vite/client">
<link rel="modulepreload" as="script" crossorigin="" href="/_nuxt/Users/foo/bar/node_modules/nuxt/dist/app/entry.mjs">
<meta charset="utf-8">
...
</head>
```

Want.
```
<head>
<meta charset="utf-8">
<link rel="modulepreload" as="script" crossorigin="" href="/_nuxt/@vite/client">
<link rel="modulepreload" as="script" crossorigin="" href="/_nuxt/Users/foo/bar/node_modules/nuxt/dist/app/entry.mjs">...
...
</head>
```

**Comment by harlan-zw** [comment]
Are you using SPA mode?

**Reply by Vivi-Ornitier** [reply]
@harlan-zw 
Yes, I using SPA mode.

**Reply by harlan-zw** [reply]
This is a bug unfortunately https://github.com/nuxt/framework/issues/5492

A fix will be out in the next week hopefully

**Reply by harlan-zw** [reply]
You can follow the advice in that issue to get around the issue for the time being

**Reply by Vivi-Ornitier** [reply]
@harlan-zw 

Has this issue been resolved in rc.13?

**Reply by harlan-zw** [reply]
It has not, there was a quick fix for it but am building for much better SPA / SSR support so will aim for next RC

**Reply by harlan-zw** [reply]
This is shipped in the edge channel
